src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,70 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+interface Props {
+  key: string;
```

Comment:
Props should not include `key` as a prop type. It's a special prop used by React for lists and is not passed to the components.